lib/cosmos/interfaces/serial_interface.rb

```diff
@@ -61,9 +62,20 @@ def connect
         @parity,
         @stop_bits,
         @write_timeout,
-        @read_timeout
+        @read_timeout,
+        @flow_control
       )
       super()
     end
+
+    # Supported Options
+    # FLOW_CONTROL - Flow control method NONE or RTSCTS. Defaults to NONE
+    def set_option(option_name, option_values)
+      super(option_name, option_values)
+      case option_name.upcase
+      when 'FLOW_CONTROL'
```

Comment:
Yes.  This is an uncommon option so that's fine.  Will just need to put a note somewhere.  Maybe in the description of the interface.